core/src/main/java/io/grpc/internal/CensusStatsModule.java

```diff
@@ -329,7 +330,9 @@ public void outboundMessage(int seqNo) {
       this.parentCtx = checkNotNull(parentCtx);
       this.startCtx =
           module.tagger.toBuilder(parentCtx)
-          .put(RpcMeasureConstants.RPC_METHOD, TagValue.create(fullMethodName)).build();
+          .put(RpcMeasureConstants.RPC_METHOD, TagValue.create(fullMethodName))
+          .put(RpcMeasureConstants.GRPC_CLIENT_METHOD, TagValue.create(fullMethodName))
```

Comment:
You probably want to create the TagValue once and re-use it.

Reply:
Done.